documentation/app/components/list-item/index.hbs

```diff
@@ -396,4 +396,32 @@
       />
     </L.ActionDropdown>
   </Cut::ListItem>
+
+  {{#each this.serviceListData as |item|}}
```

Comment:
I know this page is somewhat temporary, but could you add a margin to the list items so that we can see the elevation properly.